scripts/create-kubestellar-demo-env.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
+# Copyright 2024 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Deploys Kubestellar environment for demo purposes.
+
+set -e
+
+echo -e "Checking that pre-req softwares are installed..."
+curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v0.23.1/hack/check_pre_req.sh | bash -s -- -V kflex ocm helm kubectl docker kind
+
+output=$(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v0.23.1/hack/check_pre_req.sh | bash -s -- -A -V kflex ocm helm kubectl docker kind
+)
```

Comment:
TODO: consider curl-to-bash here.

Also: how is the user going to invoke _this_ script (create-kubestellar-demo-env.sh)? How will this script invoke the corresponding _version_ of check_pre_req.sh?

Reply:
The script can be invoke through this command once its in the pr goes through: bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v0.23.1/scripts/create-kubebestellar-demo-env.sh).

Regarding the version of check_pre_req.sh, I don't see any indications of version in the check_pre_req.sh script.

Reply:
My point is that scripts should reference each other consistently. If a user invokes version X of this script, it should in turn invoke version X of check_pre_requ.sh.